pages/documentation/developer/repositories/powsybl-open-loadflow.md

```diff
@@ -7,12 +7,10 @@ The PowSyBl Open Load Flow [repository](https://github.com/powsybl/powsybl-open-
 ## Features
 
 - a load flow solver
-- a security analysis (beta version)
 
 ## Getting started
 
 - [Guide 1]() - Run a load flow computation
-- [Guide 2]() - Run a security analysis
```

Comment:
Why do you remove that ?

Reply:
I'm sorry but I don't see any change on open loadflow file in this PR at the end.